frontend/src/entities/configuration/configuration.tsx

```diff
@@ -66,13 +66,13 @@ export function Configuration () {
                             label: 'Dynamic Configuration',
                             id: 'dynamic-config',
                             content: <DynamicConfiguration/>,
-                            disabled: true
+                            disabled: false
```

Comment:
We can just delete these, not sure we will ever need to have these again since it defaults to false.

Reply:
I actually didnt mean to include this change. I think @LightSeekerSC was going to add that in his PR

Reply:
Agreed